File: d66-social/src/main/java/com/eroldmr/d66/appuser/dto/RegisterRequest.java
package com.eroldmr.d66.appuser.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author Mompati 'Patco' Keetile
 * @created 19-12-2022 @ 12:09
 */
@Data @AllArgsConstructor @NoArgsConstructor
public class RegisterRequest {
  private String email;
  private String firstName;
  private String lastName;
  private String username;
  private String password;
}

File: d66-social/src/main/kotlin/com/eroldme/d66/user/register/dto/RegisterRequest.kt
package com.eroldme.d66.user.register.dto;

/**
 * @author Mompati 'Patco' Keetile
 * @created 19-12-2022 @ 10:10
 */
data class RegisterRequest(
  val email: String,
  val username: String,
  val password: String
)
